bears/general/DuplicateFileBear.py

```diff
@@ -0,0 +1,36 @@
+import filecmp
+import itertools
+import os.path
+
+from coalib.bears.GlobalBear import GlobalBear
+from coalib.results.Result import Result
+
+
+class DuplicateFileBear(GlobalBear):
+    LANGUAGES = {'All'}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Duplicates'}
+
+    def run(self):
+        """
+        Checks for Duplicate Files using filecmp module
+        of Standard Library
+        """
+
+        unique_tuples = [fileTuple for fileTuple in
+                         itertools.combinations(self.file_dict, 2)]
+
+        if not unique_tuples:
+            yield Result(self, 'You included no files '
+                         'or only a single file.')
+        else:
+
+            for fileTuple in unique_tuples:
+                if filecmp.cmp(fileTuple[0], fileTuple[1]):
+                    firstFileName = os.path.basename(fileTuple[1])
+                    secondFileName = os.path.basename(fileTuple[1])
+                    message = 'File ' + firstFileName + ' identical' \
```

Comment:
please never use `\` (line continuation) -- use round brackets, which will then require you align the parts in brackets correctly.